app_test.go

```diff
@@ -100,6 +100,52 @@ func TestInvokes(t *testing.T) {
 	})
 }
 
+func TestError(t *testing.T) {
+	t.Run("NilErrorOption", func(t *testing.T) {
+		var invoked bool
+
+		app := fxtest.New(t,
+			Error(nil),
+			Invoke(func() { invoked = true }),
+		)
+		err := app.Err()
+		require.NoError(t, err)
+		assert.True(t, invoked)
+	})
+
+	t.Run("SingleErrorOption", func(t *testing.T) {
+		var invoked bool
+
+		app := fxtest.New(t,
+			Error(errors.New("module failure")),
+			Invoke(func() { invoked = true }),
+		)
+		err := app.Err()
+		require.Error(t, err)
+		assert.Contains(t, err.Error(), "module failure")
+		assert.False(t, invoked)
+	})
+
+	t.Run("MultipleErrorOption", func(t *testing.T) {
+		type A struct{}
+		panicFn := func(A) { panic("do not call me") }
+
+		app := fxtest.New(t,
+			Provide(func() A { return A{} }),
+			Invoke(panicFn),
```

Comment:
Can we leave a comment here explaining that this invocation shouldn't be executed?